sysdump/sysdump.go

```diff
@@ -180,6 +186,34 @@ func Collect(c *k8s.Client, o Options) error {
 		}
 		return nil
 	})
+	// Saves the probe data in the zip file
+	errs.Go(func() error {
+		oldStdOut := os.Stdout
+		reader, writer, err := os.Pipe()
+		if err != nil {
+			return err
+		}
+		os.Stdout = writer
+		err = probe.PrintProbeResult(c, probe.Options{
+			Namespace: o.Namespace,
+			Full:      o.Full,
+			Output:    o.Output,
+			GRPC:      o.GRPC,
+		})
+		if err != nil {
+			return err
+		}
+		writer.Close()
+		os.Stdout = oldStdOut
+		out, _ := io.ReadAll(reader)
+		// This is necessary to beautify the terminal output
+		ansiEscapePattern := `\x1b\[[0-9;]*m`
```

Comment:
Rather than doing this, there might be value in adding a `karmor probe --nocolor` option?

Now I see the reason to change the `color` -> to escape sequence .. so that here the escape sequences can be removed. Is this right?

Reply:
Yes, we require it to remove some absurd characters from file!

Reply:
So is the need is to use this pattern in a flag `nocolor` and then using this flag here?

Reply:
@Manik2708  with `nocolor` flag we do not add any color or bold text, we print normal text and thus there will be no requirement for handling ANSI escape codes. In sysdump we set the [nocolor here](https://github.com/kubearmor/kubearmor-client/pull/449/files#diff-e9c40a8e7876ce0237b03aba3f2b4f81ad70f07ae0b8ff1cffa57e76ef5eec8fR196) to `true`.
does that sounds good?
(No styling is needed through sysdump)

Reply:
So, how will we remove all colors? Is there any package? Or we have to refactor?

Reply:
I don't think there is any official package for this, any other package will probably do the same what we have in the pr, refactor is the only option I see here.

Reply:
Ok, I will start working on refactoring that! Thanks!

Reply:
let's have a print func and pass the print statement and `option.noColor` to it.

Reply:
Instead of a separate `noColor`  option should we add a new output type?  let's say `o.output == "sysdump"`. wdyt @Prateeknandle @nyrahul @DelusionalOptimist

Reply:
yes, @Aryan-sharma11 that'll be good to have, so output can take values `text, nocolor-text, json`. @Manik2708 please consider this.

Reply:
That means `output` flag will take these values and give output accordingly! Am I right?

Reply:
yes